src/hotspot/cpu/x86/assembler_x86.cpp

```diff
@@ -11420,7 +11420,7 @@ void Assembler::evpabsw(XMMRegister dst, KRegister mask, Address src, bool merge
   InstructionMark im(this);
   assert(VM_Version::supports_avx512bw() && (vector_len == AVX_512bit || VM_Version::supports_avx512vl()), "");
   InstructionAttr attributes(vector_len, /* vex_w */ false,/* legacy_mode */ false, /* no_mask_reg */ false,/* uses_vl */ true);
-  attributes.set_address_attributes(/* tuple_type */ EVEX_FVM,/* input_size_in_bits */ EVEX_NObit);
+  attributes.set_address_attributes(/* tuple_type */ EVEX_FV,/* input_size_in_bits */ EVEX_NObit);
```

Comment:
This should have remained as EVEX_FVM.

Reply:
Correct, 
It does not have an embedded broadcasting variant.
https://www.felixcloutier.com/x86/pabsb:pabsw:pabsd:pabsq#:~:text=EVEX.512.66.0F38.WIG%201D%20/r%20VPABSW%20zmm1%20%7Bk1%7D%7Bz%7D%2C%20zmm2/m512

Good.